packages/grafana-ui/src/components/Logs/LogRowMessage.tsx

```diff
@@ -52,6 +53,15 @@ const getStyles = stylesFactory((theme: GrafanaTheme) => {
   };
 });
 
+function isJson(str: string) {
+  try {
+    JSON.parse(str);
+  } catch (e) {
+    return false;
+  }
+  return true;
+}
+
```

Comment:
Consider moving this function next to https://github.com/grafana/grafana/blob/abb1c107767e8011b44078f1212ce59cf718374c/public/app/core/utils/explore.ts#L183

or maybe use that function?

Reply:
Didn't know there was such a function, thank you for the tip! I'll use that one instead